### PR TITLE
Added pagination to account for missing PRs.

### PR DESCRIPTION
**Pull request by deekoder**
Some PRs were missing. Added pagination to retrieval.

### CODE REVIEW

**Review by ebozduman — APPROVED** [review]
LGTM

File: github-pull-requests.go [hunk 1]
```diff
@@ -42,8 +42,12 @@ type GitPR struct {
 
 // Retrieve the relevant pull request data from an owner/repo.
 func getPullRequests(owner string, repo string) (pullRequests []GitPR, err error) {
-	prs, _, err := buzzClient.PullRequests.List(ctx, owner, repo, nil)
-
+	opt := &github.PullRequestListOptions{
+		ListOptions: github.ListOptions{
+			PerPage: 50,
```
Comment (ebozduman) [inline github-pull-requests.go:47]: You might want to consider defining a constant for the `PerPage` value `50` and use that constant here.

**Review by kannappanr — APPROVED** [review]
LGTM

File: github-pull-requests.go (hunk 1 above)
Comment (deekoder) [reply github-pull-requests.go:47]: will do.

**Comment by deekoder** [comment]
I added a PAGINATION_COUNT constant.

**Pull request merged**